ui/src/layouts/log_in/copy_seed_words.rs

```diff
@@ -24,20 +32,18 @@ pub fn Layout(cx: Scope, page: UseState<AuthPages>, seed_words: UseRef<String>)
         });
     }
 
-    let words = use_future(cx, (), |_| {
-        to_owned![seed_words];
-        async move {
-            let mnemonic = warp::crypto::keypair::generate_mnemonic_phrase(
-                warp::crypto::keypair::PhraseType::Standard,
-            )
-            .into_phrase();
-
-            seed_words.set(mnemonic.clone());
+    let words = use_future(cx, (), |_| async move {
```

Comment:
Just as a note, warp does generate a mnemonic phrase if the `passphrase` field is `None`, in which case you can access it after the account creation in `IdentityProfile::passphrase`